lib/includes/init.js

```diff
@@ -10,6 +11,10 @@ import ObjectFactory from './ObjectFactory';
  */
 function init() {
   setObjectFactory(ObjectFactory);
+
+  // Overwrite any classes in the class registry as needed here:
+  // for example,
+  // ClassRegistry['namespace']['SomeClass'] = SomeClassExtended;
```

Comment:
Won't this file get overwritten by code generation too?  Should we warn users of that?

Does an application (like Flux) actually need to put their overrides here?  Can they put them in their own app code (E.g., a non-generated file) as long as they do it before they use any of the generated classes?

Reply:
Yeah, there's no perfect spot to put it since anything included in the exporter will be overwritten the next time the code gets re-generated. You're right that the overrides can be put pretty much anywhere as long it's invoked prior to using the classes.  What if I just delete the notes here in `init.js` and add a note to the other generated classes explaining the recommended approach?